Files/ViewModels/FolderSettingsViewModel.cs

```diff
@@ -35,7 +33,7 @@ public FolderLayoutModes LayoutMode
                 if (SetProperty(ref LayoutPreference.LayoutMode, value, nameof(LayoutMode)))
                 {
                     SetLayoutInformation();
-                    UpdateLayoutPreferencesForPath(associatedInstance.FilesystemViewModel.WorkingDirectory, LayoutPreference);
+                    LayoutPreferencesUpdateRequired?.Invoke(this, EventArgs.Empty);
```

Comment:
Pass `LayoutPreference` in EventArgs here and in other places :)

Reply:
@gave92 ?

Reply:
Yep I swear xD

![image](https://user-images.githubusercontent.com/9673091/110256618-193c3780-7f9a-11eb-8c00-371f0241890d.png)

Reply:
I mean to pass `LayoutPreference`, or better yet `FolderSettingsViewModel` to EventArgs (instead of `EventArgs.Empty`) to receive the instance in `ModernShellPage.FolderSettings_LayoutPreferencesUpdateRequired` [line](https://github.com/files-community/Files/pull/3861/files#diff-6c1efd358a0f5dcaa97e6f9742e137c49d703014d4d3a39d7da0e02e928ab41bR206) from EventArgs

Reply:
@gave92 I think you misunderstood me 😅

Reply:
Err that's your other comment

Reply:
>  mean to pass LayoutPreference, or better yet FolderSettingsViewModel to EventArgs

Could you confirm wether 393cdc4 does what you suggested?